src/blueapi/service/main.py

```diff
@@ -304,6 +304,7 @@ def get_tasks(
 @start_as_current_span(TRACER, "task.task_id")
 def set_active_task(
     task: WorkerTask,
+    instrument_session: str,
```

Comment:
Should: For `PUT`/`POST` requests you should include the parameters in the request body rather than the URL.

(see https://www.moesif.com/blog/technical/api-development/REST-API-Design-Best-Practices-for-Rest-API-Query-Parameters-and-Query-String-Usage/)